src/index.html

```diff
@@ -1,40 +1,33 @@
 <!DOCTYPE html>
-<html>
-  <head lang="en">
-    <script type="module" src="index.js" defer></script>
-    <script type="module" src="progressBar.js" defer></script>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <div class="container">
-      <svg class="logo" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="100" height="100" viewBox="0,0,256,256">
-        <g fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter"
-          stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none"
-          font-size="none" text-anchor="none" style="mix-blend-mode: normal">
-          <g transform="scale(5.12,5.12)">
-            <path
-              d="M45.27344,2.32422c-0.1875,-0.20703 -0.45703,-0.32422 -0.73828,-0.32422h-39.07031c-0.28125,0 -0.55078,0.11719 -0.73828,0.32422c-0.19141,0.20703 -0.28516,0.48438 -0.25781,0.76563l3.51953,39.42578c0.03516,0.41406 0.32422,0.75781 0.72266,0.875l16.01172,4.57031c0.08594,0.02734 0.17969,0.03906 0.27344,0.03906c0.09375,0 0.18359,-0.01172 0.27344,-0.03906l16.02344,-4.57031c0.39844,-0.11719 0.68359,-0.46094 0.72266,-0.875l3.51563,-39.42578c0.02734,-0.28125 -0.06641,-0.55859 -0.25781,-0.76562zM36.84766,15.91797h-18.8125l0.44922,5.08984h17.91016l-1.34375,15.04297l-10.05859,3.03906l-0.09766,-0.03125l-9.94141,-3.01172l-0.54297,-6.12891h4.87109l0.21094,2.37891l5.55859,1.16406l5.45703,-1.16406l0.58203,-6.4375h-17.04297l-1.32422,-14.80469h24.55859z">
-            </path>
-          </g>
-        </g>
-      </svg>
-      <h1>Semantic HTML tags</h1>
-      <h2>How many do you know?</h2>
-      <form class="form">
-        <input type="text" id="input1" title="input1" />
-        <button class="button" type="submit">Submit</button>
-      </form>
-      <div>
-        <div id="progress-live" aria-live="polite" class="visually-hidden"></div>
-        <label for="progress-bar">
-          <span id="progress-title" aria-hidden="true">Your Progress: </span>
-          <span id="progress-description" class="visually-hidden">Your progress: 0 out of 124 tags added</span>
-        </label>
-        <progress id="progress-bar" min="0" max="100" value="0" </progress>
-      </div>
-      <ul id="list">
-        <li><b>Inputs</b></li>
-      </ul>
+<html lang="en">
+
+<head>
+  <script type="module" src="index.js" defer></script>
+  <script type="module" src="progressBar.js" defer></script>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="container">
+    <img class="logo" src="/src/assets/logo.svg" alt="" />
+    <h1>Semantic HTML tags</h1>
+    <h2>How many do you know?</h2>
+    <form class="form">
+      <input class="input" type="text" id="input" />
```

Comment:
In `index.js` there are still two instances of `input1` inside the `submitVal` function. Right now we can't submit any input because of that. Once that's fixed, everything else works great.